rust_dev_preview/cross_service/rest_ses/tests/api/work_item/mocked.rs

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+//! End-to-end tests for the Rest WorkItem app with mocked AWS resources.
+//! This uses MockServer to hand-craft replies, and is run in normal tests.
+//! It additionally tests many error cases, by having the mocked AWS resources respond with errors.
+use wiremock::{matchers::any, Mock, ResponseTemplate};
+
+use crate::{
+    startup::spawn_app_mocked,
+    work_item::{fake_description, fake_guide},
+};
+
+/// Send JSON to the create endpoint, mock a failing call to RDS, and expect a 500
```

Comment:
/// Send JSON to the create endpoint, mock a failing call to Amazon Relational Database Service (Amazon RDS), and expect a 500.